tests/harness/cases/maps.proto

```diff
@@ -22,3 +22,14 @@ message MapValues { map<string, string> val = 1 [(validate.rules).map.values.str
 
 message MapKeysPattern   { map<string, string> val = 1 [(validate.rules).map.keys.string.pattern = "(?i)^[a-z0-9]+$"]; }
 message MapValuesPattern { map<string, string> val = 1 [(validate.rules).map.values.string.pattern = "(?i)^[a-z0-9]+$"]; }
+
+message MapCombination {
+    map<string, string> val = 1 [
+            (validate.rules).map = {
+                    min_pairs: 0,
+                    max_pairs: 128,
+                    keys: {string: {min_len: 2, max_len:63, pattern: "^[a-z0-9A-Z._-]$"}},
+                    values: {string: {min_len: 2, max_len:63, pattern: "^[a-z0-9A-Z._-]$"}}
+            }
+        ];
+}
```

Comment:
Can you add test cases that use this message in https://github.com/envoyproxy/protoc-gen-validate/blob/master/tests/harness/executor/cases.go? While compilation alone will be sufficient for go, others might not be detectable until their actual use.

Reply:
Sorry, I didn't mean to delay for so long.. It's been eventful last a couple of month for me. I will update the MR this weekend.

Reply:
Added test cases in `tests/harness/executor/cases.go`.